content/user/auth_phpbb.php

```diff
@@ -1,19 +0,0 @@
-<?php
-	//used to authenticate users for phpbb
```

Comment:
Let's make sure the `cmubuggy` PhpMyAdmin login still works after landing this! I checked it on the dev site and it seems fine, but just to be safe!

Reply:
Will do, I expect it does though, since it auths directly to the database.